### PR TITLE
Update cdx importer and  FdaSubmission model

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: jhipster-jdl.jdl [hunk 1]
```diff
@@ -291,7 +291,6 @@ entity FdaSubmission {
     decisionDate Instant
 
     description TextBlob
-    platform String
```
Comment (calvinlu3) [inline jhipster-jdl.jdl:294]: In `MetaImporter`, there's still some outdate code using `setPlatform` on FdaSubmission.

In `fda_submissions.csv` from data folder, the platform can be removed

File: src/test/java/org/mskcc/oncokb/curation/web/rest/CompanionDiagnosticDeviceResourceIT.java
Comment (calvinlu3) [inline src/test/java/org/mskcc/oncokb/curation/web/rest/CompanionDiagnosticDeviceResourceIT.java:1]: `findAllWithEagerRelationships` does not exist

File: src/test/java/org/mskcc/oncokb/curation/web/rest/GeneResourceIT.java [hunk 2]
```diff
@@ -190,6 +201,24 @@ void getAllGenes() throws Exception {
             .andExpect(jsonPath("$.[*].hgncId").value(hasItem(DEFAULT_HGNC_ID)));
     }
 
+    @SuppressWarnings({ "unchecked" })
```
Comment (calvinlu3) [inline src/test/java/org/mskcc/oncokb/curation/web/rest/GeneResourceIT.java:204]: `findAllWithEagerRelationships` does not exist

File: jhipster-jdl.jdl (hunk 1 above)
Comment (calvinlu3) [reply jhipster-jdl.jdl:294]: In `FdaSubmissionCriteria`, there's still platform field that can be removed

File: src/main/java/org/mskcc/oncokb/curation/importer/CdxImporter.java [hunk 3]
```diff
@@ -178,70 +176,70 @@ private CompanionDiagnosticDevice parseCdxNameColumn(String columnValue) throws
...
     }
 
-    private CancerType parseCancerTypeColumn(String columnValue) throws IllegalArgumentException {
-        columnValue = columnValue.split(",")[0].trim();
-        Optional<CancerType> optionalCancerType = cancerTypeService.findOneBySubtypeIgnoreCase(columnValue);
-        if (optionalCancerType.isPresent()) {
-            return optionalCancerType.get();
+    private List<CancerType> parseCancerTypesColumn(String columnValue) throws IllegalArgumentException {
+        List<CancerType> cancerTypes = new ArrayList<>();
+        for (String cancerType : columnValue.split("\\|")) {
```
Comment (calvinlu3) [inline src/main/java/org/mskcc/oncokb/curation/importer/CdxImporter.java:209]: I assume you modified the column in CDx sheet to use pipe when there were multiple CT?

Comment (zhx828) [reply src/main/java/org/mskcc/oncokb/curation/importer/CdxImporter.java:209]: Yup.

**Review by calvinlu3 — APPROVED** [review]
LGTM

**Pull request merged**